docker/sandbox-bundled/manifests/complete.yaml

```diff
@@ -448,6 +448,7 @@ data:
         - container
         - sidecar
         - k8s-array
+        - agent-service
```

Comment:
we need to update the agent service config as well. 
```  agent-service:
    defaultAgent:
      endpoint: "dns:///flyteagent.flyte.svc.cluster.local:8000"
      insecure: true

Reply:
No problem!
Thanks a lot for the review!